fullscreen.bs

```diff
@@ -396,6 +396,15 @@ could be an open <{dialog}> element.
  <a>simple fullscreen document</a>, then set <var>doc</var> to <var>topLevelDoc</var> and
  <var>resize</var> to true.
 
+ <li><p>If <var>doc</var>'s <a>fullscreen element</a> is not connected:
+  <ol>
+   <li><p><a for=set>Append</a> (<code>fullscreenchange</code>, <var>doc</var>'s
+   <a>fullscreen element</a>) to <var>doc</var>'s
+   <a>list of pending fullscreen events</a>.
+
+   <li><p><a lt="unfullscreen a document">Unfullscreen <var>doc</var></a>.
```

Comment:
Yes the fullscreenelement won't change synchronously (and that is the point of this change).

Reply:
`document.fullscreenElement` would change synchronously, because the element would be removed from top layer right after this code runs and before the `element.remove()` call returns.